dlc/src/main/scala/org/bitcoins/dlc/DLCMessage.scala

```diff
@@ -2,28 +2,234 @@ package org.bitcoins.dlc
 
 import org.bitcoins.core.crypto.{ECPublicKey, ExtPublicKey, Sha256DigestBE}
 import org.bitcoins.core.currency.Satoshis
-import org.bitcoins.core.protocol.Bech32Address
+import org.bitcoins.core.number.UInt32
+import org.bitcoins.core.protocol.BlockStamp.BlockTime
 import org.bitcoins.core.protocol.transaction.{
   TransactionOutPoint,
   TransactionOutput
 }
+import org.bitcoins.core.protocol.{Bech32Address, BlockStamp, NetworkElement}
+import org.bitcoins.core.psbt.InputPSBTRecord
+import org.bitcoins.core.psbt.InputPSBTRecord.PartialSignature
+import org.bitcoins.core.util.{CryptoUtil, Factory}
 import org.bitcoins.core.wallet.fee.SatoshisPerVirtualByte
+import scodec.bits.ByteVector
+import ujson._
+
+import scala.annotation.tailrec
+import scala.collection.mutable
 
 sealed trait DLCMessage
 
 object DLCMessage {
+
+  // TODO: will need to be changed when this is standardized
+  def calcEventId(
+      oracleInfo: OracleInfo,
+      contractInfo: ContractInfo,
+      timeouts: DLCTimeouts): String = {
```

Comment:
Should probably return one of `Sha256Digest` or `Sha256DigestBE` and let users call `_.hex` (one reason being cause `String` is nasty to return and another being that way they know if it is LE or BE